tests/unittests/test_handler/test_handler_reset_rmc.py

```diff
@@ -0,0 +1,39 @@
+from cloudinit.config import cc_reset_rmc
+
+from cloudinit import cloud
+from cloudinit import helpers
+from cloudinit import util
+
+from cloudinit.tests import helpers as t_help
+
+import logging
+import os
+
+LOG = logging.getLogger(__name__)
+
+
+class TestRsctNodeFile(t_help.FilesystemMockingTestCase):
```

Comment:
neither of the 2 test files/classes have methods named test_*.  So neither actually run anything when your run tox.

    $ ./tools/tox-venv  py3 - tests/unittests/test_handler/test_handler_refresh_rmc_and_interface.py tests/unittests/test_handler/test_handler_reset_rmc.py 
    inside tox:py3 running: python3 -m pytest --durations 10 tests/unittests/test_handler/test_handler_refresh_rmc_and_interface.py tests/unittests/test_handler/test_handler_reset_rmc.py
    ============================= test session starts ==============================
    platform linux -- Python 3.8.5, pytest-6.0.1, py-1.9.0, pluggy-0.13.1
    rootdir: /home/smoser/src/cloud-init, configfile: tox.ini
    plugins: cov-2.10.1
    collected 0 items                                                              

    ============================ no tests ran in 0.07s =============================

Reply:
Addressed the changes and now it is running the test cases

```./tools/tox-venv  py3 - tests/unittests/test_handler/test_handler_reset_rmc.py
inside tox:py3 running: python3 -m pytest --durations 10 tests/unittests/test_handler/test_handler_reset_rmc.py
================================================================ test session starts =================================================================
platform linux -- Python 3.6.8, pytest-6.1.1, py-1.9.0, pluggy-0.13.1
rootdir: /root/aman/cloud-init, configfile: tox.ini
plugins: cov-2.10.1
collected 2 items
tests/unittests/test_handler/test_handler_reset_rmc.py ss                                                                                                           [100%]
================================================================ slowest 10 durations ================================================================
(6 durations < 0.005s hidden.  Use -vv to show these durations.)
================================================================= 2 skipped in 0.09s =================================================================```